package.json

```diff
@@ -13,10 +13,12 @@
   },
   "scripts": {
     "build": "lage transpile types build bundle",
+    "build:selfhost": "node packages/lage/dist/lage.js transpile types build bundle",
     "watch": "lage transpile isolatedTypes --no-cache --verbose --unstable-watch",
     "change": "beachball change",
     "checkchange": "beachball check",
     "ci": "lage transpile types build test lint bundle",
+    "ci:selfhost": "node packages/lage/dist/lage.js transpile types build test lint bundle",
```

Comment:
`build` actually includes transpile, types, and bundle... so it's likely redundant here.